.travis.yml

```diff
@@ -19,8 +19,10 @@ matrix:
         - php: 7.2
         - php: 7.3
           env: SYMFONY_PHPUNIT_VERSION=7.2
+        - php: 7.4snapshot
         - php: nightly
     allow_failures:
+        - php: 7.4snapshot
```

Comment:
let's remove this line, we don't use allow_failures because everybody ignores unenforced jobs + it's green